changelog.d/15846.misc

```diff
@@ -0,0 +1 @@
+Split out 2022 changes from the changelog so the rendered version in GitHub doesn't timeout as much.
```

Comment:
> I think this probably wants to target the release branch?

Just an ongoing maintenance thing. Not really specific to this release. Going to opt for `develop`

(it does render sometimes)

Reply:
hmm, I updated it to merge into the release branch in any case. Will merge the release into `develop` after as well so we can benefit in the mean-time.